.github/templates/windows_ci_workflow.yml.j2

```diff
@@ -121,6 +139,7 @@ jobs:
           if-no-files-found: error
           name: ${{ env.BUILD_ENVIRONMENT }}
           path: C:\${{ github.run_id }}\build-results
+      !{{ wait_and_kill_ssh() }}
```

Comment:
This is an experiment with jinja2 macros, not entirely sure I like it since it makes it a bit harder to read but I guess they work 🤷🏽‍♂️